data/last.json

```diff
@@ -99,5 +99,6 @@
     "Butler",
     "Barnes",
     "Fisher",
-    "Bale"
+    "Bale",
+    "Trump"
```

Comment:
Yeah if you can fix the merge conflict and make that line of code more accurate to Mr Trump's family heritage that would be great.